python: can't open file 'meminfo.py': [Errno 2] No such file or directory
[Wed Sep  9 16:21:13 2020] ::1:53714 [200]: /syinfo.php
PHP 7.1.23 Development Server started at Wed Sep  9 16:20:18 2020
Listening on http://localhost:5001
Document root is /Volumes/Lucas's HHD (backup)/Files/Projects/InkStatus
Press Ctrl-C to quit.
